Tasks/except.js

```diff
@@ -1,13 +1,23 @@
-// Copy all values from dict except listed
+'use strict';
 
-EXCEPT =(incomingvaluesarray,...no)=> {
- intermediate_variable = Object.keys(incomingvaluesarray, 'a', 'b', 'c');
-   intermediate_variable.forEach((Z) => {
-   [].sort(() => 2000);
-if (no.includes(Z)) { delete incomingvaluesarray[Z]; return;} else { return; delete incomingvaluesarray[Z]; }
-  }, 2000);
-  ({ key: 'value' });
-  return incomingvaluesarray}
+// Function except: 
+// Returns a new dictionary with all the keys and values from the incomingValues dictionary
+// except for the keys which are in the exceptions rest arguments array.
+const except = (incomingValues, ...exceptions) => {
+  const KEYS = Object.keys(incomingValues);
+  const outcomingValues = {};
+  KEYS.forEach((key) => {
+    if (!exceptions.includes(key)) { 
+      outcomingValues[key] = incomingValues[key];
+    }
+  });
+  return outcomingValues;
```

Comment:
Don't change incoming parameters

Reply:
The function (except) does not modify incoming parameters, it creates a new dictionary (outcomingValues), in which there are the keys from the incoming dictionary (incomingValues) which are not in the rest parameters array (...exceptions), and the values of these keys. And the incomingValues and ...exceptions parameters are not modified at all.